spec/models/embed/was_seed_thumbs_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.